docker-compose-dev.yml

```diff
@@ -29,16 +29,20 @@ web:
     - DEBUG_TB_INTERCEPT_REDIRECTS=False
     - INVENIO_APP_CONFIG_ENVS=CACHE_REDIS_HOST,CFG_DATABASE_HOST,DEBUG,DEBUG_TB_INTERCEPT_REDIRECTS
   volumes:
-    - ./invenio:/code/invenio:ro
-    - ./docs:/code/docs:rw
-    - ./scripts:/code/scripts:ro
+    - ../invenio/invenio:/code/invenio:ro
+    - ../invenio/docs:/code/docs:rw
+    - ../invenio/scripts:/code/scripts:ro
+    - ./invenio_demosite:/code-overlay/invenio_demosite:ro
     - /code/invenio/base/translations
+    - /home/invenio
+    - /tmp:/tmp:rw
+    - /usr/local/var/invenio.base-instance
```

Comment:
No, these are already part of the image, see https://github.com/inveniosoftware/invenio/blob/maint-2.0/Dockerfile#L138-L141